src/vm/frames.h

```diff
@@ -418,6 +418,10 @@ class Frame : public FrameBase
 
 public:
 
+#if defined(FEATURE_PAL) && !defined(DACCESS_COMPILE) 
```

Comment:
different compilation units with different compilation flags would see different definition of the class Frame. In case of field members, it will cause different size of object.
Ifdefing definition of virtual methods might change the order of virtual functions in a virtual table and its size.
See http://en.wikipedia.org/wiki/One_Definition_Rule#Summary third point.